.github/workflows/flow-build-application.yaml

```diff
@@ -43,34 +43,29 @@ defaults:
     shell: bash
 
 jobs:
-  code:
```

Comment:
This should not be removed.

Reply:
it is empty.  All it does it checkout the code and do an `npm ci`, it doesn't actually compile anything.

Reply:
code-style is a dependency and will do the same